README.md

```diff
@@ -1,2 +1,22 @@
-hello world
+# The Team 7 Project (Name TBA)
 
+A website that connects the homeless to shelter.
+
+## Running Locally
+
+Redirect to ./App and run:
+
+```bash
+mvn package exec:java
```

Comment:
I can't think of any reason why it wouldn't, but did you double-check that this actually works?

Reply:
Just tripled checked and the project opens on port 8080 👍